src/main/resources/application.properties

```diff
@@ -26,7 +26,6 @@ management.health.diskspace.threshold=1073742000
 
 # metrics
 management.statsd.metrics.export.flavor=datadog
-management.metrics.web.server.request.autotime.enabled=false
```

Comment:
the migration guide mentions "Should be applied at the ObservationRegistry level.". Do we need to do that?

Reply:
Do you have a reference to this ? I just saw that it's been removed.

Reply:
https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-3.0.0-RC1-Configuration-Changelog#removed-in-300-rc1

Reply:
More info here: https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-3.0-Migration-Guide#micrometer-and-metrics-changes and here: https://github.com/spring-projects/spring-boot/issues/41745

Overall I think this is OK though I don't recall why we turned that metric off.